### PR TITLE
Fixes of HypVINN & options for conform that disable critieria

### PR DESCRIPTION
**Pull request by dkuegler**
- Some fixes to the HypVINN pipeline 
  - formatting/doc
  - naming of HypoVINN->HypVINN (unified)
  - fix some errors with default hypvinn file names and arguments
  - generally improve and clean up the hypvinn interface and functions
- conform.py gets new flags `--no_lia`, `--no_strict_lia`, `--no_img_size`, `--no_vox_size` which deactivate parts of the conform script:
  - `no_lia`: rotation of the affine is not touched, image may remain RAS
  - `no_strict_lia`: rotation of the affine is adapted, but only towards LIA such that no interpolation would be necessary, but will also be LIA (never RAS)
  - `no_img_size` no padding/cropping of the image
  - `no_iso_vox`~`no_vox_size`~ no harmonization of voxel size, for example in combination with `no_strict_lia`, the image is not made iso. voxelsize
-  Cleanup of shell scripts
  -  Remove extra "2&>1" in shell scripts in echo statements (they never have stderr output)

### CODE REVIEW

File: FastSurferCNN/data_loader/conform.py [hunk 1]
```diff
@@ -107,16 +129,42 @@ def options_parse():
...
+    )
+    advanced.add_argument(
+        "--no_lia",
+        dest="force_lia",
+        action="store_false",
+        help="Ignore the reordering of data into lia (without interpolation). "
+             "Superceeds --no_strict_lia",
+    )
+    advanced.add_argument(
+        "--no_vox_size",
```
Comment (m-reuter) [inline FastSurferCNN/data_loader/conform.py]: maybe better no_iso_vox ?? or no_vox_iso ?

File: FastSurferCNN/data_loader/conform.py [hunk 2]
```diff
@@ -107,16 +129,42 @@ def options_parse():
...
-        help="Specifies the target data type of the target image or 'any' (default: 'uint8', "
-        "as in FreeSurfer)",
+        help="Specifies the target data type of the target image or 'any' (default: "
+             "'uint8', as in FreeSurfer)",
+    )
+    advanced.add_argument(
+        "--no_strict_lia",
+        dest="force_strict_lia",
+        action="store_false",
+        help="Ignore the forced lia reorientation.",
```
Comment (m-reuter) [inline FastSurferCNN/data_loader/conform.py]: use "LIA" instead of "lia" also in line 154.

File: FastSurferCNN/data_loader/conform.py [hunk 3]
```diff
@@ -107,16 +129,42 @@ def options_parse():
...
+    advanced.add_argument(
+        "--no_lia",
+        dest="force_lia",
+        action="store_false",
+        help="Ignore the reordering of data into lia (without interpolation). "
+             "Superceeds --no_strict_lia",
+    )
+    advanced.add_argument(
+        "--no_vox_size",
+        dest="force_vox_size",
```
Comment (m-reuter) [inline FastSurferCNN/data_loader/conform.py]: also here, maybe better force_iso_vox

File: FastSurferCNN/data_loader/conform.py (hunk 1 above)
Comment (dkuegler) [reply FastSurferCNN/data_loader/conform.py]: I guess that is also true and way easier to understand

File: FastSurferCNN/data_loader/conform.py [hunk 4]
```diff
@@ -107,16 +129,42 @@ def options_parse():
...
+        action="store_false",
+        help="Ignore the forced isometric voxel size (depends on --conform_min).",
+    )
+    advanced.add_argument(
+        "--no_img_size",
```
Comment (m-reuter) [inline FastSurferCNN/data_loader/conform.py:164]: why not no_img_dim ?

Comment (dkuegler) [reply FastSurferCNN/data_loader/conform.py:164]: dim is usually the number of dimensions, so usually 3 or 4 in python (numpy/torch)
shape would be what we mean, but I am not sure it that is more "helpful"

File: FastSurferCNN/data_loader/conform.py [hunk 5]
```diff
@@ -560,33 +661,57 @@ def conform(
...
+        The epsilon for the affine check.
+
+    Returns
+    -------
+    bool
+        The result.
+    """
+    _v2v = np.abs(vox2vox[:3, :3])
+    # check 1: have exactly 3 times 1/-1 rest 0, check 2: all 1/-1 or 0
+    return abs(_v2v.sum() - 3) > eps or np.any(np.maximum(_v2v, abs(_v2v - 1)) > eps)
```
Comment (m-reuter) [inline FastSurferCNN/data_loader/conform.py:701]: shouldn't this be abs(_v2v.sum()) - 3 ? and also abs(_v3v) - 1 ?

Comment (dkuegler) [reply FastSurferCNN/data_loader/conform.py:701]: v2v is already abs in line 699

File: recon_surf/recon-surf.sh [hunk 6]
```diff
@@ -531,16 +531,16 @@ if [[ ! -f "$mdir/transforms/talairach.lta" ]] || [[ ! -f "$mdir/transforms/tala
   echo " " 2>&1 | tee -a $LF
   echo "============= Computing Talairach Transform ============" 2>&1 | tee -a $LF
   echo " " 2>&1 | tee -a $LF
```
Comment (m-reuter) [inline recon_surf/recon-surf.sh]: Here (and maybe other places) the 2>&1 can also be removed

File: FastSurferCNN/data_loader/conform.py (hunk 4 above)
Comment (m-reuter) [reply FastSurferCNN/data_loader/conform.py:164]: yes, I think size is fine

**Pull request merged**